src/fileio.cpp

```diff
@@ -42,6 +42,7 @@ static const char * const _subdirs[] = {
 	"save" PATHSEP "autosave" PATHSEP,
 	"scenario" PATHSEP,
 	"scenario" PATHSEP "heightmap" PATHSEP,
+	"scenario" PATHSEP "towndata" PATHSEP,
```

Comment:
Just as a FYI, if you want to be able to load heightmaps + towndata, they can't be in different folders. How the content system works, is that 1 content piece needs everything in a single folder.

But I also like your idea of having this for SE-only, that people need to load towndata on top of their scenario / heightmap. But just the headsup.

Reply:
Ah, thanks for noting this. Something to consider as we move forward with this and consider bundling town data with heightmaps or other info.

Reply:
My rationale for doing towns and heightmaps separately instead of bundling them in a .tar or similar, is that the heightmap import often doesn't go perfectly the first try. The max height is wrong, or some coasts need cleanup, etc. Loading towns takes a lot of time, so it would be frustrating to force them to happen simultaniously.

For now, this format is for scenario creators, not a full "reproducible scenario" format for use by the average player.

That still doesn't answer the question of where to put the files. I guess we could always move them later.

Reply:
I've moved town data into the heightmap directory. I don't plan to bundle anything in this PR, but this will make it easier later.